packages/hoprd/src/commands/utils/peers.ts

```diff
@@ -0,0 +1,25 @@
+import type HoprCoreConnector from '@hoprnet/hopr-core-connector-interface'
+import type Hopr from '@hoprnet/hopr-core'
+import type PeerId from 'peer-id'
+import { isBootstrapNode } from './isBootstrapNode'
+
+/**
+ * Get node's peers.
+ * @returns an array of peer ids
+ */
+export function getPeers(
```

Comment:
Do we actually need this anymore? The function definition is longer than the code..